python/ql/src/experimental/semmle/python/frameworks/NoSQL.qll

```diff
@@ -141,11 +110,10 @@ private module NoSql {
    * mongo.db.user.find({'name': safe_search})
    * ```
    *
-   * `mongo.db.user.find` would be a collection method, and so the result.
+   * `mongo.db.user.find` would be a collection method.
    */
-  private DataFlow::Node mongoCollectionMethod() {
-    mongoCollection() = result.(DataFlow::AttrRead).getObject() and
-    result.(DataFlow::AttrRead).getAttributeName() instanceof MongoCollectionMethodNames
+  private API::Node mongoCollectionMethod() {
```

Comment:
we don't seem to need this predicate any more. Can you inline it to `MongoCollectionCall`? :blush:

Reply:
Done :-)